mlir-tensorrt/compiler/test/python/mlir_tensorrt_runtime/test_create_memref.py

```diff
@@ -414,7 +415,7 @@ def memref_alloc():
 test_released_internally()
 
 # CHECK-LABEL: Test memref is released internally with an external reference
```

Comment:
Looks like we should also update the naming/commentary of this test to "not released internally"?